README.md

```diff
@@ -79,21 +79,21 @@ We're just getting started!
 
 The criteria for becoming a core contributor is to participate at least once
 per month over a period of 3 months.  Participation could include providing
-feedback in the weekly D&I meetings, providing feedback on docs, or making
-other contributions on GitHub (commits / issues).  People not participating
+feedback in the weekly VMG meetings, providing feedback on docs, or making
+other contributions on GitHub (commits/issues).  People not participating
 over a 3 month period may be removed as core contributors.
 
 If you'd like to be on our squad, an easy way to start is by going through the
 [issue list](https://github.com/chaoss/wg-value/issues) and fixing some. :tada:
 
 **All Contributors**
 
-Ordered by first name
+Ordered by the first name
```

Comment:
I disagree with the edit in line 91. "the" is not needed here

Reply:
Ok.